src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/vcf/VcfFuncotationFactoryUnitTest.java

```diff
@@ -165,19 +165,11 @@ private VariantContext createVariantContext(final String contig,
                                         Allele.create("C"), FACTORY_NAME)
                         )
                 ),
-                // Three overlapping VCF features:
+                // No overlapping VCF features, since there are no indels in dbSNP (the test datasource), so the ground truth should be a default entry, which was constructed here manually:
```

Comment:
Could you add in a test case here that has 3 overlapping features?

Reply:
I clarified the comment.  Three overlap by position... none overlap if you include ref and alt in the "overlap".

Done.

Reply:
(I'm assuming that you do not want me to create a VCF datasource with three identical records).